Sources/BKMoneyKit/BKCardNumberFormatter.m

```diff
@@ -27,7 +27,7 @@ - (instancetype)init
     self = [super init];
     if (self) {
         
-        NSString *filePath = [[NSBundle bundleForClass:[self class]] pathForResource:@"BKMoneyKit.bundle/CardPatterns" ofType:@"plist"];
+        NSString *filePath = [SWIFTPM_MODULE_BUNDLE pathForResource:@"CardPatterns" ofType:@"plist"];
```

Comment:
This will probably break CocoaPods support, right? Should we remove the podspec together with this PR?

Reply:
good shout, will do that

Reply:
✅